packages/types/definitions.ts

```diff
@@ -286,7 +286,7 @@ export type Scenario = {
       /**
        * @title HTTP engine
        */
-      engine: 'http';
+      engine?: 'http';
```

Comment:
I'm still surprised this worked 🤔 We need to test this carefully but so far automated tests confirm the intention as valid.